specification/dataprotection/resource-manager/readme.md

```diff
@@ -46,6 +46,15 @@ semantic-validator: true
 message-format: json
 ```
 
+### Tag: package-2021-01
+
+These settings apply only when `--tag=package-2021-01-01` is specified on the command line.
+
+``` yaml $(tag) == 'package-2021-01-01'
```

Comment:
You also need to update this one to package-2021-01

Reply:
fixed this